### PR TITLE
Adds debug launch configuration and task for Slint

### PR DESCRIPTION
**Pull request by SuchAFuriousDeath**

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -40,13 +40,30 @@
...
       },
+      "args": [
+        "--kernel",
+        "${workspaceFolder}/build/obkrnl"
+      ],
       "cwd": "${workspaceFolder}"
     },
     {
-      "name": "Kernel",
+      "name": "Kernel (Qt)",
```
Comment (ultimaweapon) [inline .vscode/launch.json:50]: The parentheses here does not look good on the VS Code due to VS Code also put some text after this with parentheses. Better to change to another format (e.g. `Kernel | Qt`).

File: .vscode/launch.json [hunk 2]
```diff
@@ -40,13 +40,30 @@
...
+      "targetCreateCommands": [
+        "target create ${workspaceFolder}/build/obkrnl",
+        "target modules load --file ${workspaceFolder}/build/obkrnl -s 0xffffffff82200000"
+      ],
+      "processCreateCommands": [
+        "gdb-remote 1234"
+      ]
+    },
+    {
+      "name": "Kernel (Slint)",
```
Comment (ultimaweapon) [inline .vscode/launch.json:63]: Same here.

**Review by ultimaweapon — CHANGES_REQUESTED** [review]

File: gui/src/main.rs [hunk 3]
```diff
@@ -183,21 +234,30 @@ enum ApplicationError {
     #[error("failed to run wizard")]
     RunWizard(#[source] slint::PlatformError),
 
+    #[error("get current executable path")]
+    GetCurrentExe(#[source] std::io::Error),
+
```
Comment (ultimaweapon) [inline gui/src/main.rs:239]: This does not seems right.

**Review by ultimaweapon — CHANGES_REQUESTED** [review]

File: gui/src/main.rs (hunk 3 above)
Comment (SuchAFuriousDeath) [reply gui/src/main.rs:239]: What about it?

Comment (SuchAFuriousDeath) [reply gui/src/main.rs]: Done

**Review by ultimaweapon — APPROVED** [review]

**Pull request merged**